### DISCUSSION TITLE
exporting data with string columns

### DISCUSSION THREAD

**Discussion by rootsmusic** [DaxStudio/DaxStudio#817] [Q&A] [answered]
In the data model, a column is in text format because its values have leading "0" characters.  In DAXStudio's Export Data Wizard, I'm selecting to "quote all string fields".  But why aren't the column's values being quoted in the exported .csv?  Thanks.

**Comment by dgosbell** [comment, marked as answer]
I'm not able to reproduce this issue. When you hover over this column does the tooltip say that it is a string column?

I just created a simple example pbix file and it exported fine, quoting the string field as it should.
![image](https://user-images.githubusercontent.com/8241830/158899602-d4563fa9-811c-4e10-b4cc-9a38b9f6a012.png)

**Comment by rootsmusic** [comment]
Sorry @dgosbell, Dax Studio isn't to blame.  The following Python script reads export.csv, which is exported from Dax Studio.

> import pandas as pd
> export = pd.read_csv('/Users/admin/Desktop/export.csv', sep=',', lineterminator='\n')
> export.sort_values(by=['COUNTYFP10']).head(25)

But why is export['COUNTYFP10'] converting to number (from string format)?
![image](https://user-images.githubusercontent.com/22034168/158911345-ec8017c1-41aa-4db1-8cb1-3ccef8b01db1.png)

**Reply by dgosbell** [reply]
I don't use python all that much, but it looks like this is part of the way the read_csv function works. A quick google turned up this post https://stackoverflow.com/questions/35790267/avoid-converting-data-to-int-automatically-while-reading-using-pandas-data-frame which suggests using the dtype parameter to avoid pandas automatically casting to a numeric type.
